sycl/include/sycl/handler.hpp

```diff
@@ -945,6 +946,34 @@ class __SYCL_EXPORT handler {
       sycl::ext::oneapi::experimental::execution_scope threadScope,
       sycl::ext::oneapi::experimental::execution_scope coordinationScope);
 
+  template <typename Properties>
+  void checkAndSetClusterRange(const Properties &Props) {
+    if constexpr (ext::oneapi::experimental::hasClusterDim<Properties, 1>()) {
+      setKernelUsesClusterLaunch(true);
+      MNDRDesc.setClusterDimensions(
+          Props
+              .template get_property<
```

Comment:
Can you confirm that we need the `.template` here? Sometimes with newer compilers we don't need this

Reply:
Yes, trying without the template led to a compilation error